dvc/stage/params.py

```diff
@@ -11,12 +11,16 @@ class StageParams:
     PARAM_META = "meta"
     PARAM_ALWAYS_CHANGED = "always_changed"
     PARAM_PARAMS = "params"
+    PARAM_METRICS = "metrics"
+    PARAM_PLOTS = "plots"
 
 
 class OutputParams(Enum):
```

Comment:
Still needed?

Reply:
OutputParams? Not really, I just didn't touch them in this PR for now.